bundles/python/python_lexer.moon

```diff
@@ -100,4 +140,14 @@ howl.aux.lpeg_lexer ->
       identifier,
       decorator
     }
+
+    f_string: any f_strings
+
+    string: any {
+      f_prefix * V'f_string'
+      string
+    }
   }
+
+  rules[k] = v for k, v in pairs interpolations
```

Comment:
I'm not sure why it is necessary to inject all interpolations into the main rule set? Since `f_string` is already in `all`, wouldn't that just work?